.storybook/preview.tsx

```diff
@@ -6,6 +7,7 @@ import { OPEN_SANS_FONT, STORYBOOK_MODES, STORYBOOK_SIZES } from './constants';
 import type { Preview, ReactRenderer } from '@storybook/react';
 
 import '../styles/new/index.css';
+import { NotificationProvider } from '@/providers/notificationProvider';
```

Comment:
```suggestion
import { NotificationProvider } from '../providers/notificationProvider';
```

We avoid using `@` relative imports on Storybook